### DISCUSSION TITLE
explanation on model.print_model()

### DISCUSSION THREAD

**Discussion by nanas5150** [lululxvi/deepxde#1814] [Q&A]
Hi everyone, can someone please explain to me what is the meaning when I run this model.print_model()?
and why there is 200 data for each?

This is the net informations:
#Define Network and Hyperparameters
layer_size = [1] + [10] * 1 + [4]
activation = "tanh"
initializer = "Glorot uniform"

NN = dde.nn.FNN(layer_size, activation, initializer)
model = dde.Model(data, NN)
model.compile("adam", lr=0.001)
losshistory, train_state = model.train(iterations=1000)

**Variable: dense/kernel:0, Shape: (200,)**
[(  0,) (169,) (186,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 54,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) ( 48,) (123,) ( 48,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 80,) (123,) ( 48,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 12,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 47,) (107,) (101,) (114,) (110,)
 (101,) (108,) (  0,) (  0,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) (208,) (178,) (153,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (100,) (101,)
 (118,) (105,) ( 99,) (101,) ( 58,) ( 71,) (160,) (244,) (184,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) (192,) (244,) (184,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (192,) (244,) (184,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (247,) (185,) (101,) (  5,) (144,) ( 85,) (  0,) (  0,) ( 48,) (136,)
 (153,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 21,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense/bias:0, Shape: (200,)**
[(160,) ( 29,) (164,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (176,) ( 70,) (187,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (208,) ( 70,) (187,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 10,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 47,) ( 98,) (105,) ( 97,) (115,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) (112,) (110,) (183,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  8,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (144,) (103,) (183,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) (176,) (103,) (183,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (176,) (103,) (183,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (242,) (245,) (182,) ( 92,) (149,) ( 85,) (  0,) (  0,) (194,) (247,)
 (183,) ( 92,) (149,) ( 85,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_1_1/kernel:0, Shape: (200,)**
[(  0,) (232,) (118,) ( 90,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 21,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (112,) (198,) (186,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (224,) (156,) (186,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 16,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (161,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) ( 32,) ( 37,) (184,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (100,) (101,)
 (118,) (105,) ( 99,) (101,) ( 58,) ( 71,) ( 80,) ( 37,) (184,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) (112,) ( 37,) (184,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (112,) ( 37,) (184,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (208,) (105,) (164,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 32,) ( 84,)
 (182,) ( 92,) (149,) ( 85,) (  0,) (  0,) (105,) ( 97,) ( 98,) (108,)
 (101,) ( 79,) (112,) ( 95,) (  0,) ( 48,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_1_1/bias:0, Shape: (200,)**
[(192,) (250,) (185,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 47,) (116,) ( 97,) (115,) (107,) ( 58,)
 ( 48,) ( 47,) (240,) (247,) (139,) ( 91,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 16,) (248,) (139,) ( 91,) (149,) ( 85,)
 (  0,) (  0,) ( 14,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 49,) ( 95,) ( 49,) ( 47,)
 ( 98,) (105,) ( 97,) (115,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) ( 16,) ( 18,) ( 80,) ( 90,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (132,) (122,) (  0,) (  0,) ( 64,) ( 18,) ( 80,) ( 90,)
 (149,) ( 85,) (  0,) (  0,) ( 96,) ( 18,) ( 80,) ( 90,) (149,) ( 85,)
 (  0,) (  0,) ( 96,) ( 18,) ( 80,) ( 90,) (149,) ( 85,) (  0,) (  0,)
 (  0,) (122,) (182,) ( 92,) (149,) ( 85,) (  0,) (  0,) (240,) (163,)
 (186,) ( 92,) (149,) ( 85,) (  0,) (  0,) (109,) ( 47,) ( 82,) (101,)
 ( 97,) (100,) ( 47,) ( 82,) (  0,) ( 97,) (100,) ( 86,) ( 97,) (114,)
 (105,) ( 97,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_2/kernel:0, Shape: (200,)**
[(224,) (181,) (185,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (112,) ( 63,) (182,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (234,) (114,)
 (132,) (122,) (  0,) (  0,) (144,) ( 63,) (182,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 14,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 50,) ( 47,) (107,) (101,)
 (114,) (110,) (101,) (108,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) (224,) (201,) (183,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (115,) (244,)
 (227,) (  5,) (144,) ( 85,) (  0,) (  0,) ( 16,) (202,) (183,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) ( 48,) (202,) (183,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 48,) (202,) (183,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 ( 48,) ( 75,) (237,) ( 27,) (132,) (122,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  1,) (  0,) (  0,) (  0,)
 (144,) ( 85,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_2/bias:0, Shape: (200,)**
[( 32,) (138,) (153,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (109,) (101,) (  0,) (105,) ( 99,) (101,)
 (115,) (  0,) ( 48,) (165,) (153,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 80,) (165,) (153,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 12,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 50,) ( 47,) ( 98,) (105,)
 ( 97,) (115,) (  0,) ( 85,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) ( 96,) (138,) (153,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (144,) (187,)
 (186,) ( 92,) (149,) ( 85,) (  0,) (  0,) (112,) (110,) ( 91,) ( 91,)
 (149,) ( 85,) (  0,) (  0,) (144,) (110,) ( 91,) ( 91,) (149,) ( 85,)
 (  0,) (  0,) (144,) (110,) ( 91,) ( 91,) (149,) ( 85,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (132,) (122,) (  0,) (  0,) ( 81,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (144,) (187,) (186,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) (  0,) (124,) (234,) (114,) (132,) (122,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_3/kernel:0, Shape: (200,)**
[( 64,) ( 38,) (184,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 97,) (108,) (104,) (111,) (115,) (116,)
 ( 47,) (114,) (176,) ( 69,) (182,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) ( 58,) ( 48,)
 (  0,) (122,) (  0,) (  0,) (208,) ( 69,) (182,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 14,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 51,) ( 47,) (107,) (101,)
 (114,) (110,) (101,) (108,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) (128,) (126,) (181,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 64,) ( 82,) (185,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) ( 96,) ( 82,) (185,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 96,) ( 82,) (185,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_3/bias:0, Shape: (200,)**
[(160,) ( 82,) (185,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 48,) ( 75,) (237,) ( 27,) (132,) (122,)
 (  0,) (  0,) (176,) (222,) (171,) ( 91,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (208,) (222,) (171,) ( 91,) (149,) ( 85,)
 (  0,) (  0,) ( 12,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 51,) ( 47,) ( 98,) (105,)
 ( 97,) (115,) (  0,) (  0,) (  0,) (  0,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) ( 48,) (169,) (184,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 48,) ( 86,) (183,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) ( 80,) ( 86,) (183,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 80,) ( 86,) (183,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 ( 16,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 32,) (112,)
 (182,) ( 92,) (149,) ( 85,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (254,) (183,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_4/kernel:0, Shape: (200,)**
[( 48,) (152,) (184,) ( 92,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 64,) (249,) (185,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (240,) ( 89,) (175,) ( 91,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (175,) ( 91,)
 (149,) ( 85,) (  0,) (  0,) ( 16,) ( 90,) (175,) ( 91,) (149,) ( 85,)
 (  0,) (  0,) ( 14,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 52,) ( 47,) (107,) (101,)
 (114,) (110,) (101,) (108,) (  0,) (199,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) (208,) ( 90,) (175,) ( 91,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (101,) (112,)
 (108,) (105,) ( 99,) ( 97,) ( 58,) ( 48,) (240,) (175,) (185,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) ( 16,) (176,) (185,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 16,) (176,) (185,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 ( 64,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 96,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 53,) (227,) (250,) (  2,)
 (144,) ( 85,) (  0,) (  0,) (  0,) ( 25,) (187,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Variable: dense_4/bias:0, Shape: (200,)**
[( 96,) ( 77,) (180,) ( 90,) (149,) ( 85,) (  0,) (  0,) ( 44,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) ( 44,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (112,) ( 63,) (151,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (  9,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (108,) (111,)
 ( 99,) ( 97,) (108,) (104,) (111,) (115,) (116,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (144,) ( 63,) (151,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 12,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (100,) (101,) (110,) (115,) (101,) ( 95,) ( 52,) ( 47,) ( 98,) (105,)
 ( 97,) (115,) (  0,) (105,) (122,) (101,) (229,) (121,) (  9,) (254,)
 ( 34,) ( 81,) ( 60,) (242,) (144,) (176,) (182,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) ( 18,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 ( 30,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) ( 19,) (175,)
 (227,) (  5,) (144,) ( 85,) (  0,) (  0,) (208,) (215,) (185,) ( 92,)
 (149,) ( 85,) (  0,) (  0,) (240,) (215,) (185,) ( 92,) (149,) ( 85,)
 (  0,) (  0,) (240,) (215,) (185,) ( 92,) (149,) ( 85,) (  0,) (  0,)
 (100,) (101,) (118,) (105,) ( 99,) (101,) ( 58,) ( 71,) ( 80,) ( 85,)
 ( 58,) ( 48,) (  0,) (122,) (  0,) (  0,) (128,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)
 (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,) (  0,)]

**Comment by lululxvi** [comment]
https://github.com/lululxvi/deepxde/blob/d0c5aa25414ca469e07564d973e1ee499aff3c13/deepxde/model.py#L1108

**Reply by nanas5150** [reply]
Thank you for your reply, I did try to look at the code previously, despite my beginner level with python, I don't quite understand the content of deepxde/deepxde/model.py
appreciate if you or anybody can give me simple explanation for this.

**Reply by lululxvi** [reply]
print tf.trainable_variables()
